Scripts/Python/ki/__init__.py

```diff
@@ -3716,6 +3716,21 @@ def IsContentMutable(self, nodeRef):
                         return False
         return True
 
+    ## Determines whether the content Node Reference is deletable (and by extension movable to other folders).
+    def IsContentDeletableAndMovable(self, nodeRef):
+
+        # Get its parent folder.
+        if isinstance(nodeRef, ptVaultNodeRef):
+            folder = self.BKCurrentContent.getParent()
+            item = self.BKCurrentContent.getChild()
+            if folder and item:
+                if folder := folder.upcastToFolderNode():
+                    # Saver ID of zero should mean a DRC item. These inbox items are the only kind
+                    # of content that should *not* be able to be deleted or moved
+                    if folder.folderGetType() == PtVaultStandardNodes.kGlobalInboxFolder and self.BKCurrentContent.getSaverID() == 0:
```

Comment:
Sometimes, items in the global inbox might have a player sender. Better to simply reject anything in the global inbox.

```suggestion
                    if folder.folderGetType() == PtVaultStandardNodes.kGlobalInboxFolder:
```